client/app/dedicatedUniverseComponents/bytes/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
+
+import ducBytesFilter from './bytes.filter';
+
+const moduleName = 'ducBytes';
+
+angular
+  .module(moduleName, [
+    translate,
+    translateAsyncLoader,
+  ])
+  .filter('ducBytes', ducBytesFilter)
+  .run(/* @ngTranslationsInject ./translations */);
```

Comment:
Nothing ?

Reply:
Related to: https://github.com/ovh-ux/component-rollup-config#translation-inject